plugins/readtimer.koplugin/main.lua

```diff
@@ -10,6 +11,9 @@ local T = require("ffi/util").template
 local ReadTimer = WidgetContainer:extend{
     name = "readtimer",
     time = 0,  -- The expected time of alarm if enabled, or 0.
+    dialog = nil,
+    last_interval_time = 0,
+    timer_way = nil
```

Comment:
It can be just true/false, no need of strings.

Reply:
done